ScalingCarousel/Classes/ScalingCarouselView.swift

```diff
@@ -20,7 +20,7 @@ open class ScalingCarouselView: UICollectionView {
     // MARK: - Properties (Private)
 
     private var lastCurrentCenterCellIndex: IndexPath?
-
+    public var scrollDirection: UICollectionView.ScrollDirection = .horizontal
```

Comment:
To match the access level of other 'public' properties, can we change this to `open`, allowing it to be overridden if someone wants to do that

Also, minor, but can we move this down into the `MARK: - Properties (Public)` section below.

Reply:
Hi @superpeteblaze,
I made it open and moved it under the MARK - PUBLIC properties.  
Thank you for pointing out minor things as well